bin/proving-service/src/main.rs

```diff
@@ -103,18 +103,20 @@ mod test {
 
         let transaction_witness = TransactionWitness::from(executed_transaction);
 
-        let request_1 = Request::new(ProveTransactionRequest {
-            transaction_witness: transaction_witness.to_bytes(),
+        let request_1 = Request::new(ProveRequest {
+            proof_type: 0,
```

Comment:
Could we make this stronger-typed so we can write something like `ProofType::Transaction` instead?

Reply:
Sure, it is already implemented within the generated files. I'm replacing it with: `ProofType::Transaction.into()`.